src/anybody.js

```diff
@@ -28,6 +28,81 @@ function intersectsButton(button, x, y) {
   )
 }
 
+const PAUSE_BODY_DATA = [
+  {
+    bodyIndex: 0,
+    seed: '0x34f645e62a9fb47226675e36c76e717bee1cc4688cdbd2ce1a2343e0fb210afa',
```

Comment:
don't think seed is necessary anymore, we pick random face based on `this.day` now. At the same time I think it's ok for starting faces to be totally random. Maybe just remove seed since it's a little confusing why it's there?

Reply:
cool! i just put it there bc it was on the body data i console.logged

Reply:
removed seeds 🌱